docker/ci/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM rust:1.53-slim
```

Comment:
So we have two directories where it could make sense to host it: `ci` and `docker/ci`.
I think it makes more sense to do under `ci`, like the other stuff we use with the workflows.
Especially if this is meant to be an image just for the audit job.

`./docker` is more intended for the community users who want to bake their own jormungandr images.